ethers-ext/example/v6/browser-html/main.js

```diff
@@ -230,3 +255,121 @@ async function sendFeeDelegatedSC() {
     };
   });
 }
+
+async function signAndSendGaslessTxs() {
+  try {
+    // ------- before swap -------
+    const balanceOfABI = ["function balanceOf(address owner) view returns (uint256)"];
+    const testToken = new ethers.Contract(testTokenAddr, balanceOfABI, provider);
+
+    const kaiaBeforeSwap = await provider.getBalance(accounts[0].address);
+    const tokenBeforeSwap = await testToken.balanceOf(accounts[0].address);
+
+    $("#kaiaBeforeSwap").html(`${ethers_ext.formatKaia(kaiaBeforeSwap)}`);
+    $("#tokenBeforeSwap").html(`${ethers_ext.formatKaia(tokenBeforeSwap)}`);
+
+    const testTokenToSwap = document.getElementById('testTokenSwapAmount')
+    const testTokenToSwapBN = BigInt(ethers_ext.parseKaia(testTokenToSwap.value))
+
+    // ------- prepare transactions -------
+    const signer = await provider.getSigner(accounts[0].address);
+
+    const network = await signer.provider.getNetwork();
+    const chainId = Number(network.chainId);
+
+    const gsr = ethers_ext.gasless.getGaslessSwapRouter(provider, chainId);
+
+    // ------- send approve -------
+    const maxUint256 = BigInt("0xffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff")
+    let approveTx = await ethers_ext.gasless.getApproveTx(
+      provider,
+      accounts[0].address,
+      testTokenAddr,
+      gsr.address,
+      maxUint256
+    );
+
+    if (approveTx) {
+      approveTx['type'] = 0;
```

Comment:
This logic should be in `getApproveTx()` and `getSwapTx()`

Reply:
I agree with you. 
I'll submit a patch including changing the Approve Amount to maxUint256.